File: forge/src/main/java/com/hazzabro124/marionetta/forge/VRPlugin.java
package com.hazzabro124.marionetta.forge;

import net.blf02.forge.VRAPIPlugin;
import net.blf02.forge.VRAPIPluginProvider;
import net.blf02.vrapi.api.IVRAPI;

@VRAPIPlugin
public class VRPlugin implements VRAPIPluginProvider {
    public static IVRAPI vrAPI = null;

    @Override
    public void getVRAPI(IVRAPI ivrapi) {
        vrAPI = ivrapi;
        VRPluginStatus.hasPlugin = true;
    }
}

File: common/src/main/kotlin/org/valkyrienskies/tournament/VRPlugin.kt
package org.valkyrienskies.tournament

import net.blf02.forge.VRAPIPlugin
import net.blf02.forge.VRAPIPluginProvider
import net.blf02.vrapi.api.IVRAPI

@VRAPIPlugin
class VRPlugin : VRAPIPluginProvider {
    companion object {
        var vrAPI: IVRAPI? = null
    }

    override fun getVRAPI(ivrapi: IVRAPI) {
        vrAPI = ivrapi
        VRPluginStatus.hasPlugin = true
    }
}